.travis.yml

```diff
@@ -1,8 +1,8 @@
-sudo: required
-language: bash
-
-services:
-  - docker
+language: go
```

Comment:
can try using `gimme` manually, but travis is not set up to do multi language projects.